Dockerfile

```diff
@@ -30,30 +30,44 @@ RUN yum -y --setopt=tsflags=nodocs update && \
            /var/opennms/reports && \
     mkdir -p /opennms-data/logs \
              /opennms-data/rrd \
-             /opennms-data/reports && \
+             /opennms-data/mibs \
+             /opennms-data/reports \
+             /opt/opennms-etc-overlay \
+             /opt/opennms/assets && \
+    mv /var/opennms/mibs/compiled /opennms-data/mibs/ && \
+    rm -rf /var/opennms/mibs && \
     ln -s /opennms-data/logs /opt/opennms/logs && \
     ln -s /opennms-data/rrd /var/opennms/rrd && \
-    ln -s /opennms-data/reports /var/opennms/reports
+    ln -s /opennms-data/mibs /var/opennms/mibs && \
```

Comment:
Just out of curiosity, what is the "mibs" folder for?

Reply:
The MIBs folder contains, the default "compiled" MIBs we provide (to avoid dependency issues, and cycling dependency problems). Also, that will contain the MIBs a given customer wants to compile. If we don't track that directory and someone spend time adding a bunch of MIBs, that work will be lost. Of course, not all the work is lost if there are event definitions and data collection config already generated.

That said, maybe it would be a good enhancement moving the `mibs` folders to `/opt/openms/etc` in the future.